docs/technical_reference/costing/electroNP.rst

```diff
@@ -9,51 +9,51 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
 .. csv-table::
    :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "parameter_name", "1", ":math:`\text{dimensionless}`"
+   "Hydraulic retention time", ":math:`HRT`", "HRT", "1.3333", ":math:`\text{hr}`"
+   "Reactor sizing cost", ":math:`C_V`", "sizing_cost", "1000", ":math:`\text{$/m^3}`"
+   "Magnesium chloride cost", ":math:`C_{MgCl2}`", "magnesium_chloride_cost", "0.0786", ":math:`\text{$/kg}`"
+   "Phosphorus recovery value*", ":math:`C_{RP}`", "phosphorus_recovery_value", "-0.07", ":math:`\text{$/kg}`"
```

Comment:
Units not displaying correctly

Reply:
Also, we should decide on whether we want to just display `$` or `USD_YYYY` as the units for cost. I've done the latter in my documentation so far, but whatever we decide to do, we should be consistent.